docs/api/general_rest_usage.md

```diff
@@ -38,44 +38,49 @@ The API will reply with:
 HTTP/1.1 200 OK
 Accept-Patch: application/vnd.ez.api.ContentUpdate+xml;charset=utf8
 Content-Type: application/vnd.ez.api.ContentInfo+xml
-Content-Length: xxx
 ```
 
-The length of our content, provided by the Content-Length header, isn't *that* useful.
-
 ###### HTTP Code
 
-The API responded here with a standard 200 OK HTTP response code, which is the expected response code for a "normal" GET request. Some GET requests, like [getting a Content item's current version](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#13241%C2%A0%C2%A0%C2%A0get-current-version), may reply with a 301 Moved permanently, or 307 Temporary redirect code.
+The API responded here with a standard `200 OK` HTTP response code, which is the expected response code for a "normal" GET request. Some GET requests, like [getting a Content item's current version](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#13241%C2%A0%C2%A0%C2%A0get-current-version), may reply with a `301 Moved permanently`, or `307 Temporary redirect` code.
+
+Errors are indicated with HTTP error codes, like `404 Not Found`, or `500 Internal Server Error`. The [REST specifications](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst) provide the list of every HTTP response code you can expect from implemented resources.
 
-Errors will be indicated with HTTP error codes, like 404 for Not Found, or 500 for Internal server error. The [REST specifications](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst) provide the list of every HTTP response code you can expect from implemented resources.
+###### Content Type header
```

Comment:
Not sure about removing the hyphen here and in Accept Patch below, maybe header names should be repeated verbatim, as they appear in code?